pageserver/src/tenant/mgr.rs

```diff
@@ -244,15 +248,64 @@ pub async fn shutdown_all_tenants() {
         }
     };
 
+    // Set tenant (and its timlines) to Stoppping state.
+    // Since we can only transition into Stopping state after activation is complete,
+    // run it in a JoinSet so all tenants have a chance to stop before we git SIGKILLed.
```

Comment:
```suggestion
    // Set tenant (and its timlines) to Stoppping state.
    //
    // Since we can only transition into Stopping state after activation is complete,
    // run it in a JoinSet so all tenants have a chance to stop before we get SIGKILLed.
```